script/release.sh

```diff
@@ -139,13 +139,20 @@ CONTRIBUTORS=$(git log "${LAST_RELEASE_VERSION}".. --format="%aN" --reverse | so
 CHANGELOG=$(git log "${LAST_RELEASE_VERSION}".. --oneline | grep -v 'Merge pull request')
 
 CHECKSUM=""
-for file in $(ls bin/docker-machine*); do
+for file in $(ls bin/docker-machine-*); do
   SHA256=$(openssl dgst -sha256 < "${file}")
   MD5=$(openssl dgst -md5 < "${file}")
   LINE=$(printf "\n * **%s**\n  * sha256 \`%s\`\n  * md5 \`%s\`\n\n" "$(basename ${file})" "${SHA256}" "${MD5}")
   CHECKSUM="${CHECKSUM}${LINE}"
 done
 
+# Generate Checksum file
+(
+    cd bin
+    shasum -a 256 docker-machine-* > ../sha256sum.txt
```

Comment:
Can you please use `openssl dgst` here instead?  It should be present on both OSX and Linux and `md5sum` is not present on my OSX box.